Cauli/Configuration/Configuration.swift

```diff
@@ -25,6 +25,9 @@ import Foundation
 /// The Configuration is used to configure Cauli at initialization time.
 /// The `Configuration.standard` is a sensibly chosen configuration set.
 public struct Configuration {
+    ///
+    public typealias PrePersistHook = ((Record) -> Record)
```

Comment:
I am interested in your comments regarding the naming of this callback.

Reply:
I think I would chose a situation independent name for the typealias. Something like `RecordModifier` or such.